internal/cli/atlas/alerts/settings/create.go

```diff
@@ -116,11 +137,13 @@ func CreateBuilder() *cobra.Command {
 	cmd.Flags().StringVar(&opts.notificationWebhookURL, flag.NotificationWebhookURL, "", usage.NotificationWebhookURL)
 	cmd.Flags().StringVar(&opts.notificationWebhookSecret, flag.NotificationWebhookSecret, "", usage.NotificationWebhookSecret)
 	cmd.Flags().StringSliceVar(&opts.notificationRoles, flag.NotificationRole, []string{}, usage.NotificationRole)
+	cmd.Flags().StringVarP(&opts.filename, flag.File, flag.FileShort, "", usage.AlertConfigFilename)
+
+	_ = cmd.MarkFlagFilename(flag.File)
```

Comment:
Could you update the e2e test to test this flag? This will also allow us to have an example of file to use for the command to share with the doc team for the documeantion